components/alert/alert-toast.js

```diff
@@ -22,6 +22,8 @@ class AlertToast extends LitElement {
 			open: { type: Boolean, reflect: true },
 			subtext: { type: String },
 			type: { type: String, reflect: true },
+			_hasFocus: { type: Boolean },
+			_hasMouse: { type: Boolean },
```

Comment:
If you make these properties that Lit is tracking, then it'll cause `render()` to be called when they change. I don't _think_ you need that, so just exclude them from here and set them in the constructor like you already are and you should be good.